assets/css/glimesh/components/stream.scss

```diff
@@ -24,4 +24,37 @@
         overflow-y: auto;
         width: 100%;
     }
+}
+
+@media (max-width: 768px) {
+    .btn-responsive {
+        padding: 2px 4px;
+        line-height: 1;
+        border-radius: 3px;
+    }
+    .f_txt {
```

Comment:
What do you think about replacing these with a common class you can add to each element you want to hide? 

Optionally Bootstrap provides these classes we can use to achieve the same effect with the Bootstrap best practices included: https://getbootstrap.com/docs/4.6/utilities/display/#hiding-elements

Reply:
That looks way better, going to try that now, thank you!

Reply:
Looks awesome!